docs/api/channels.rst

```diff
@@ -0,0 +1,328 @@
+.. SPDX-License-Identifier: MIT
+
+.. currentmodule:: disnake
+
+Channels
+========
+
+This section documents everything related to channels and threads.
+
+Classes
```

Comment:
a lot of the classes in this category are discord models, except for PartialMessageable.